backend/EasyMeets.Core/EasyMeets.Core.BLL/MappingProfiles/TeamMemberProfile.cs

```diff
@@ -11,7 +11,7 @@ public TeamMemberProfile()
             CreateMap<TeamMember, NewMeetingMemberDto>()
                 .ForMember(dest => dest.Name, opt => opt.MapFrom(src => src.User.Name))
                 .ForMember(dest => dest.Image, opt => opt.MapFrom(src => src.User.ImagePath))
-                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.User.Id));
+                .ForMember(dest => dest.Id, opt => opt.MapFrom(src => src.Id));
```

Comment:
You don need this is properties have the same name

Reply:
Removed it.